dist/css/custom.css

```diff
@@ -265,10 +265,90 @@ div.dashboard div.card-header label.form-check {
     }
 }
 
-    .navbar-dark {
-        box-shadow: inset 0 -1px 0 0 rgba(98, 105, 118, 0.16);
-    }
+.navbar-dark {
+    box-shadow: inset 0 -1px 0 0 rgba(98, 105, 118, 0.16);
+}
+
+.list-group-item-action:hover, .list-group-item-action:focus {
+    color: #000;
+}
 
-    .list-group-item-action:hover, .list-group-item-action:focus {
-        color: #000;
-    }
\ No newline at end of file
+.otc {
+	position: relative;
+	width: 100%;
+	margin: 0 auto;
+}
+
+.otc fieldset {
+	border: 0;
+	padding: 0;
+	margin: 0;
+}
+
+.otc fieldset div {
+	display: flex;
+	align-items: center;
+}
+
+.otc legend {
+	margin: 0 auto 1em;
+	color: #5555FF;
+}
+.otc input:active,
+.otc input:focus{
+    border-color: #a4cdfd;
+    outline: none;
+}
+.otc input::selection{
+    background-color: transparent;
+    color: #206bc4;
+    border-color: #a4cdfd;
+}
+
+input[type="number"] {
```

Comment:
You can not generally set the input type "number" to another style! Look what happened:

![grafik](https://github.com/PAWECOGmbH/saaster/assets/22148114/a8209b4c-2bdc-409c-83c2-811084a6e029)